orchestra/models/core/models.py

```diff
@@ -576,6 +577,34 @@ class PayRate(PayRateMixin, models.Model):
     end_date = models.DateField(null=True, blank=True)
 
 
+class ChecklistTemplate (ChecklistTemplateMixin, BaseModel):
```

Comment:
Given the objects are called Todos, can we call this TodoListTemplate and TotoListTemplateMixin?